src/coral_models/protocols.py

```diff
@@ -20,8 +20,7 @@
 
 
 class Processor(Protocol):
-    def __call__(self, *args, **kwargs) -> BatchEncoding:
-        ...
+    def __call__(self, *args, **kwargs) -> BatchEncoding: ...
```

Comment:
Why the ellipsis on the same line? Did black do this?

Reply:
Black did this.I am using black 24.2.0.